### PR TITLE
Added md that explains usage of the exported Keras JSON file

### PR DESCRIPTION
**Pull request by JPJPJPOPOP**
Added a .md file that explains the basics of using of the exported Keras JSON file and provides a worked out example.

### CODE REVIEW

**Comment by vpn1997** [comment]
You can just print the model summary rather than testing and training the  model on a dataset :)

**Comment by JPJPJPOPOP** [comment]
@vpn1997 I did want to show an example implementation.

File: tutorials/keras_json_usage.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+### Basics of Using the Exported Keras Model
+
+We want to export our model for Keras from Fabrik.
+
+1. First, select the 2nd button from the left in the Actions section of the sidebar.
+<img src="exportbutton.png">
```
Comment (utsavgarg) [inline tutorials/keras_json_usage.md]: The referenced image is not present.

File: tutorials/keras_json_usage.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+2. A drop-down list should appear. Select Keras.
+    * This should download a JSON file to your computer.
+<img src="exportdropdown.png">
```
Comment (utsavgarg) [inline tutorials/keras_json_usage.md]: The referenced image is not present.

File: tutorials/keras_json_usage.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+3. Rename the file to ```model.json```.
+
+4. Load the model from the JSON file using the following code:
+    ```
+    from keras.models import model_from_json
+
+    # Read and load the JSON file
+    json_file = open('./directory-to-file/model.json', 'r')
```
Comment (utsavgarg) [inline tutorials/keras_json_usage.md]: Change this to `json_file = open('<path_to_file>/model.json', 'r')`

File: tutorials/keras_json_usage.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+   ```
+   from keras.models import model_from_json
+   import numpy
+   import os
+
+   # Fix random seed to allow similar accuracy measures at the end
+   numpy.random.seed(7)
+
+   # Load pima indians dataset
+   dataset = numpy.loadtxt("./directory-to-file/pima-indians-diabetes.csv", delimiter=",")
```
Comment (utsavgarg) [inline tutorials/keras_json_usage.md]: Change this to `dataset = numpy.loadtxt('<path_to_file>/pima-indians-diabetes.csv', delimiter=',')`

File: tutorials/keras_json_usage.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+   # Split the dataset into input (X) and output (Y) variables
+   X = dataset[:,0:8]
+   Y = dataset[:,8]
+
+   # Load the model from JSON file
+   json_file = open('./directory-to-file/model.json', 'r')
```
Comment (utsavgarg) [inline tutorials/keras_json_usage.md]: Change this to json_file = open('<path_to_file>/model.json', 'r')

File: tutorials/keras_json_usage.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+   # Evaluate the model
+   scores = loaded_model.evaluate(X, Y, verbose=0)
+
+   # Print final accuracy
+   print("%s: %.2f%%" % (loaded_model.metrics_names[1], scores[1] * 100))
+   ```
+4. Then run the code in terminal.
+   ```
+   python ./directory-to-file/kerasJSONLoader.py
```
Comment (utsavgarg) [inline tutorials/keras_json_usage.md]: Change this to `python <path_to_file>/kerasJSONLoader.py`

File: tutorials/keras_json_usage.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+   ```
+You should be getting around 76-78% accuracy.
+
+This code basically trains and evaluates the loaded model on the dataset.
```
Comment (utsavgarg) [inline tutorials/keras_json_usage.md]: remove "basically"

File: README.md [hunk 8]
```diff
@@ -88,6 +88,8 @@ KERAS_BACKEND=theano python manage.py runserver
 * Use `example/caffe/GoogleNet.prototxt` for caffe import
 * Use `example/keras/vgg16.json` for keras import
 
+### [Usage of exported Keras JSON file](https://github.com/Cloud-CV/Fabrik/blob/master/tutorials/keras_json_usage.md)
```
Comment (utsavgarg) [inline README.md]: Change this to
```
### Documentation
* [Using a Keras model exported from Fabrik](https://github.com/Cloud-CV/Fabrik/blob/master/tutorials/keras_json_usage.md)
```

**Comment by utsavgarg** [comment]
LGTM :+1:
